README.md

```diff
@@ -7,28 +7,49 @@ Blocking ads is your choice. I lean more to the opinion of this [video](https://
 
 ## Installation
 
-Download the latest release zip [here](https://github.com/2bc4/streamlink-ttvlol/releases/latest/download/streamlink-ttvlol.zip) and extract it, then follow the instructions below.
+***Follow the instructions below for your specific operating system.***
+
+[Windows](#windows), [Linux](#linux) or [MacOS](#macos) .
+
+If you wish to install Manually. You can get the latest release [here](https://github.com/2bc4/streamlink-ttvlol/releases/latest/download/twitch.py).
 
 ### Windows:
 #### Automatically
-Run the `install.bat` file. This should automatically install the modified twitch.py file to your AppData `streamlink` folder.
+Copy the command below and paste it into powershell. ***Do not run powershell as admin.*** 
+
+If you don't know how to open powershell. Press windows key + r and type ```powershell``` into the box and press enter. Copy the contents of the box below and then right click in the powershell window and press enter.
+```
+$null= New-Item -ItemType Directory -Path .\AppData\Roaming\streamlink\plugins -Force; iwr -Uri 'https://github.com/2bc4/streamlink-ttvlol/releases/latest/download/twitch.py' -OutFile .\AppData\Roaming\streamlink\plugins\twitch.py
+```
 #### Manually
 Navigate to:
 ```
 C:\Users\%username%\AppData\Roaming\streamlink
 ```
 Then create the `plugins` folder and drop the twitch.py inside of it.
 ### Linux:
+#### Automatically
+Run the below command in terminal. 
+```
+INSTALL_DIR="${XDG_DATA_HOME:-${HOME}/.local/share}/streamlink/plugins" mkdir -p "$INSTALL_DIR"; wget -O "$INSTALL_DIR" 'https://github.com/2bc4/streamlink-ttvlol/releases/latest/download/twitch.py'
```

Comment:
Can we change this to curl? I believe curl is more ubiquitous on Linux and it will match the MacOS command. Also, I forgot a semicolon after the variable.

```suggestion
INSTALL_DIR="${XDG_DATA_HOME:-${HOME}/.local/share}/streamlink/plugins"; mkdir -p "$INSTALL_DIR"; curl -L -o "$INSTALL_DIR"/twitch.py 'https://github.com/2bc4/streamlink-ttvlol/releases/latest/download/twitch.py'
```

Reply:
Sure I don't see why not. My Manjaro install has curl and I assume its pretty common.